application/play/README.md

```diff
@@ -0,0 +1,41 @@
+<p align="center">
+  <img src="https://secrethub.io/img/integrations/play-framework/github-banner.png" alt="Play + SecretHub" height="230">
+</p>
+<br/>
+
+This Play example checks if the environment variables `DEMO_USERNAME` and `DEMO_PASSWORD` have been set. If that's the case, you'll receive a `200` on http://localhost:8080 and if it's not, you'll get a `500`.
+
+## Prerequisites
+1. [Docker](https://docs.docker.com/install/) installed and running
+1. [SecretHub](https://secrethub.io/docs/start/getting-started/#install) installed
+1. A SecretHub repo that contains a `username` and `password` secret. To create it, run `secrethub demo init`.
+
+## Running the example
+
+Set the SecretHub username in an environment variable
+```
+export SECRETHUB_USERNAME=<your-username>
+```
+
+Create a service account for the demo repo
+```
+secrethub service init --description demo_service \
+--permission read --file demo_service.cred ${SECRETHUB_USERNAME}/demo
+```
+
+Build the play docker demo
+```
+docker build . -t play-secrethub-demo
```

Comment:
Running this command in the current directory (`application/play`) will not work, because there is no `Dockerfile` here. I suggest changing it to: `docker build java -t play-secrethub-demo` or `docker build scala -t play-secrethub-demo` and mentioning that you can choose either one depending on which language you want to see in action.

Reply:
Adjusted.